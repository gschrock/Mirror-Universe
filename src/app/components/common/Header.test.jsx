import React from 'react';
import Header from './Header.jsx';
import { shallow } from 'enzyme';

describe('Header component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Header/>);
    expect(wrapper).toMatchSnapshot();
  });
});
