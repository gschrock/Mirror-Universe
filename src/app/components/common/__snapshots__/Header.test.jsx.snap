// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <header>
    <nav>
        <ul>
            <li>
                <Link
                    onlyActiveOnIndex={false}
                    style={Object {}}
                    to="/"
                >
                    Home
                </Link>
            </li>
            <li>
                <Link
                    onlyActiveOnIndex={false}
                    style={Object {}}
                    to="/about"
                >
                    About
                </Link>
            </li>
        </ul>
    </nav>
</header>,
  "nodes": Array [
    <header>
      <nav>
            <ul>
                  <li>
                        <Link
                              onlyActiveOnIndex={false}
                              style={Object {}}
                              to="/"
                        >
                              Home
                        </Link>
                  </li>
                  <li>
                        <Link
                              onlyActiveOnIndex={false}
                              style={Object {}}
                              to="/about"
                        >
                              About
                        </Link>
                  </li>
            </ul>
      </nav>
</header>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance":  {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Header />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <header>
          <nav>
                    <ul>
                              <li>
                                        <Link
                                                  onlyActiveOnIndex={false}
                                                  style={Object {}}
                                                  to="/"
                                        >
                                                  Home
                                        </Link>
                              </li>
                              <li>
                                        <Link
                                                  onlyActiveOnIndex={false}
                                                  style={Object {}}
                                                  to="/about"
                                        >
                                                  About
                                        </Link>
                              </li>
                    </ul>
          </nav>
</header>,
        "_debugID": 2,
        "_renderedOutput": <header>
          <nav>
                    <ul>
                              <li>
                                        <Link
                                                  onlyActiveOnIndex={false}
                                                  style={Object {}}
                                                  to="/"
                                        >
                                                  Home
                                        </Link>
                              </li>
                              <li>
                                        <Link
                                                  onlyActiveOnIndex={false}
                                                  style={Object {}}
                                                  to="/about"
                                        >
                                                  About
                                        </Link>
                              </li>
                    </ul>
          </nav>
</header>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Header />,
}
`;
